app/services/npq/application/accept.rb

```diff
@@ -179,13 +180,22 @@ def validate_funded_place?
       end
 
       def npq_contract
+        return unless statement
+
         @npq_contract ||=
-          NPQContract.where(
+          NPQContract.find_by(
             cohort_id: cohort.id,
             npq_lead_provider_id: npq_application.npq_lead_provider_id,
             course_identifier: npq_application.npq_course.identifier,
             version: statement.contract_version,
```

Comment:
do we need to set this to `statement&.contract_version` if the statement doesn't exist?

Reply:
So before we get the NPQContract we check the statement exists (returning if it doesn't) so we're fine assuming it will always exist here